client/src/areas/party/components/IconWithProgress.tsx

```diff
@@ -0,0 +1,57 @@
+import CircularProgress, {
+  CircularProgressProps
+} from "@material-ui/core/CircularProgress";
+import green from "@material-ui/core/colors/green";
+import * as React from "react";
+import Icon, { IconProps } from "@material-ui/core/Icon/Icon";
+import styled from "styled-components";
+
+export interface IIconWithProgressProps {
+  isLoading?: boolean;
+  title?: string;
+  icon?: string;
+  onClick: () => any;
+}
+
+export const IconWithProgress: React.FunctionComponent<
```

Comment:
this icon belongs in the common/components folder

Reply:
Moved in next commit